core/src/com/unciv/logic/civilization/Civilization.kt

```diff
@@ -356,24 +358,31 @@ class Civilization : IsPartOfGameInfoSerialization {
     fun getCompletedPolicyBranchesCount(): Int = policies.adoptedPolicies.count { Policy.isBranchCompleteByName(it) }
     private fun getCivTerritory() = cities.asSequence().flatMap { it.tiles.asSequence() }
 
-    fun getPreferredVictoryType(): String {
+    fun getPreferredVictoryTypes(): List<String> {
         val victoryTypes = gameInfo.gameParameters.victoryTypes
         if (victoryTypes.size == 1)
-            return victoryTypes.first() // That is the most relevant one
-        val victoryType = nation.preferredVictoryType
-        return if (victoryType in gameInfo.ruleset.victories) victoryType
-               else Constants.neutralVictoryType
+            return listOf(victoryTypes.first()) // That is the most relevant one
+        val victoryType: List<String> = (setOf(nation.preferredVictoryType) + setOf(getPersonality().preferredVictoryType))
```

Comment:
Why not setOf(A,B)?

Reply:
Tbh, I forget why I'm even using `setOf` instead of `listOf`

Reply:
Huh... Tbh Idk which types are exclusive like that and which wasn't. Might be a reason to switch to using a set overall instead of returning a list